src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -36,6 +37,23 @@ object LocalBackend {
     }
   }
 
+  def findTempFile(root: Path, prefix: String) = {
+    val regex = s"$prefix.*\\.tmp$$".r.unanchored
+    val filesWithPrefix = Try(Files.newDirectoryStream(root)).map(
+      stream => stream.iterator().toIterator.toSeq.collect {
```

Comment:
e.g.:

    case head :: Nil => ???
    case Nil = ???
    case head :: tail = ???

That might not be 100% correct I freetyped it but there abouts